doc/rados/configuration/mon-config-ref.rst

```diff
@@ -1138,7 +1068,7 @@ Miscellaneous
               the stored checksums with the computed ones of all the stored
               keys.
 :Type: Integer
-:Default: 3600*24
+:Default: 86400
```

Comment:
for the same reason, `3600*24` is more readable than `86400` . as one is able to tell that it's 24 hours immediately without firing up `bc` =)